### ISSUE TITLE
Add FUSE as a dependency

### ISSUE THREAD

**Issue by alanking** [jasoncoposky/mungefs#11]

**Comment by bh9** [comment]
fuse needs to be at least version 2.9.1, as `fallocate` wasn't introduced until then

**Comment by kjsanger** [comment]
Any word on this? Building `4.2.7` I found that this dependency was missing from externals and had to build it by hand. It doesn't appear to be in `versions.json`. (Should I be raising an issue in `externals`?)

**Comment by trel** [comment]
Correct, we aren't currently building fuse in externals... since all OSs have it available... but I'm not clear on the versioning needed vs. available on different OSs/versions... maybe we *do* need to add it to externals?

https://github.com/irods/mungefs/blob/master/CMakeLists.txt#L141
